libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -872,19 +877,42 @@ def DetachDisk(self, disk: 'GoogleComputeDisk') -> None:
     response = request.execute()
     self.BlockOperation(response, zone=self.zone)
 
-  def Delete(self, delete_disks: bool = False) -> None:
+  def Delete(
+      self, delete_disks: bool = False, force_delete: bool = False) -> None:
     """Delete an Instance.
 
     Args:
       delete_disks (bool): force delete all attached disks (ignores the 'Keep
-        when instance is deleted' bit).
+          when instance is deleted' bit).
+      force_delete (bool): force delete the instance, even if deletionProtection
+          is set to true.
     """
     disks_to_delete = []
     if delete_disks:
       disks_to_delete = [
           disk['source'].split('/')[-1] for disk in self.GetValue('disks')]
 
     gce_instance_client = self.GceApi().instances()
+
+    if force_delete:
+      if self.deletion_protection:
```

Comment:
```suggestion
    if force_delete and self.deletion_protection:
```

Reply:
Should we output a warning if `force_delete` is `False` and `self.deletion_protection` is `True` so the user knows to add the parameter?

Reply:
Yep, added